src/libs/OptionsListUtils.ts

```diff
@@ -532,9 +532,15 @@ function getAlternateText(option: ReportUtils.OptionData, {showChatPreviewLine =
     const report = ReportUtils.getReportOrDraftReport(option.reportID);
     const isAdminRoom = ReportUtils.isAdminRoom(report);
     const isAnnounceRoom = ReportUtils.isAnnounceRoom(report);
+    const isGroupChat = ReportUtils.isGroupChat(report);
+    const isExpenseThread = ReportUtils.isMoneyRequest(report);
 
-    if (!!option.isThread || !!option.isMoneyRequestReport) {
-        return option.lastMessageText ? option.lastMessageText : Localize.translate(preferredLocale, 'report.noActivityYet');
+    if (isExpenseThread || option.isMoneyRequestReport) {
```

Comment:
`isExpenseThread` and `option.isMoneyRequestReport` are equivalent, no?

Reply:
They are different.

`isMoneyRequestReport` checks for expense/IOU report.
https://github.com/Expensify/App/blob/05ca35c015b268e495fe94eff36c9eda7afb78bc/src/libs/OptionsListUtils.ts#L741
https://github.com/Expensify/App/blob/05ca35c015b268e495fe94eff36c9eda7afb78bc/src/libs/ReportUtils.ts#L1611-L1614

`isExpenseThread` checks if it's the transaction thread.
https://github.com/Expensify/App/blob/05ca35c015b268e495fe94eff36c9eda7afb78bc/src/libs/ReportUtils.ts#L1603-L1606

https://github.com/Expensify/App/blob/05ca35c015b268e495fe94eff36c9eda7afb78bc/src/libs/ReportUtils.ts#L1566-L1586

Reply:
Interesting, those method names are definitely misleading. Thanks for clarifying though